### PR TITLE
Feature/movie card

### PR DESCRIPTION
**Pull request by yousefo1995**

### CODE REVIEW

File: src/components/MovieCard/MovieCard/MovieCard.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { StyledCard, StyledWishlistBtn } from "./style";
+
+const MovieCard = ({
+  title = "title",
+  rate = "0.0",
+  image = "https://picsum.photos/id/870/200/300?grayscale&blur=2",
+}) => {
+  return (
+    <StyledCard>
+      <CardActionArea>
```
Comment (owaysasi) [inline src/components/MovieCard/MovieCard/MovieCard.js]: add "disableRipple" in props in CardActionArea to stop giving the ripple effect on each click on the card

File: src/components/MovieCard/MovieCard/MovieCard.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          </Link>
+        </CardContent>
+
+        <StyledWishlistBtn />
+      </CardActionArea>
+
+      <CardActions>
+        <StyledCardButton>
+          Watch now
+          <LaunchIcon fontSize="small" />{" "}
```
Comment (owaysasi) [inline src/components/MovieCard/MovieCard/MovieCard.js]: replace the "LaunchIcon" with plus Icon "+" to be look like the design

**Review by owaysasi — CHANGES_REQUESTED** [review]

File: src/components/MovieCard/MovieCard/MovieCard.js (hunk 1 above)
Comment (yousefo1995) [reply src/components/MovieCard/MovieCard/MovieCard.js]: done

File: src/components/MovieCard/MovieCard/MovieCard.js (hunk 2 above)
Comment (yousefo1995) [reply src/components/MovieCard/MovieCard/MovieCard.js]: done

**Pull request merged**